### PR TITLE
[21] Contact Form Validation

### PR DESCRIPTION
**Pull request by BenjaminKamena**
## Task: `Dom task`
- `added contact.js file`
contactValidation

### CODE REVIEW

File: submissions/benjaminKamena/Dom/signUp.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+// sign-up form validation
+
```
Comment (ToluAkin) [inline submissions/benjaminKamena/Dom/signUp.js:2]: This file exist already. Remove it from this pull request. I think you should sync your branch so you can get the file from the previous pull request @BenjaminKamena

File: submissions/benjaminKamena/JavaScript/index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
+//TASK 1
+
```
Comment (ToluAkin) [inline submissions/benjaminKamena/JavaScript/index.js:2]: Same comment as above

File: submissions/benjaminKamena/JavaScript/index.html [hunk 3]
```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
```
Comment (ToluAkin) [inline submissions/benjaminKamena/JavaScript/index.html:3]: Same comment as above

**Review by ToluAkin — CHANGES_REQUESTED** [review]

File: submissions/benjaminKamena/JavaScript/index.html (hunk 3 above)
Comment (BenjaminKamena) [reply submissions/benjaminKamena/JavaScript/index.html]: I have removed the files as requested

**Review by adaugochi — APPROVED** [review]
Some of the files you pushed already existed. Ensure you sync your branch

File: submissions/benjaminKamena/Dom/contact.js [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+        field.nextElementSibling.innerHTML = error_1;
+        event.preventDefault();
+        return false;
+    } else if(!field.value.match(regex)){
+        field.nextElementSibling.innerHTML = error_2;
+        event.preventDefault();
+        return false;
+    } else {
+        field.nextElementSibling.innerHTML = "";
+        return false;
```
Comment (adaugochi) [inline submissions/benjaminKamena/Dom/contact.js:23]: This should return true

**Review by adaugochi — APPROVED** [review]

**Pull request merged**